code/modules/projectiles/projectile/energy.dm

```diff
@@ -65,21 +75,19 @@
 	icon_state = "dragnetfield"
 	light_range = 3
 	anchored = TRUE
+	var/obj/item/device/radio/beacon/teletarget
 
 /obj/effect/nettingportal/Initialize()
 	. = ..()
-	var/obj/item/device/radio/beacon/teletarget = null
-	for(var/obj/machinery/computer/teleporter/com in GLOB.machines)
-		if(com.target)
-			if(com.power_station && com.power_station.teleporter_hub && com.power_station.engaged)
-				teletarget = com.target
 
 	addtimer(CALLBACK(src, .proc/pop, teletarget), 30)

```

Comment:
remove teletarget on here and on 85.

you are not calling the object's teletarget in the arg, you are REALLY calling for var/teletarget which defaults to null